releases/v19.2.2.md

```diff
@@ -0,0 +1,132 @@
+---
+title: What&#39;s New in v19.2.2
+toc: true
+summary: Additions and changes in CockroachDB version v19.2.2 since version v19.2.1
+---
+
+## December 16, 2019
+
+Get future release notes emailed to you:
+
+<div class="hubspot-install-form install-form-1 clearfix">
+    <script>
+        hbspt.forms.create({
+            css: '',
+            cssClass: 'install-form',
+            portalId: '1753393',
+            formId: '39686297-81d2-45e7-a73f-55a596a8d5ff',
+            formInstanceId: 1,
+            target: '.install-form-1'
+        });
+    </script>
+</div>
+
+### Downloads
+
+<div id="os-tabs" class="clearfix">
+    <a href="https://binaries.cockroachdb.com/cockroach-v19.2.2.darwin-10.9-amd64.tgz"><button id="mac" data-eventcategory="mac-binary-release-notes">Mac</button></a>
+    <a href="https://binaries.cockroachdb.com/cockroach-v19.2.2.linux-amd64.tgz"><button id="linux" data-eventcategory="linux-binary-release-notes">Linux</button></a>
+    <a href="https://binaries.cockroachdb.com/cockroach-v19.2.2.src.tgz"><button id="source" data-eventcategory="source-release-notes">Source</button></a>
+</div>
+
+### Docker image
+
+{% include copy-clipboard.html %}
+~~~shell
+$ docker pull cockroachdb/cockroach-unstable:v19.2.2
+~~~
+
+
+### General changes
+
+- Improved the consistency checker's log output. [#42149][#42149]
+- Nodes that have been terminated as the result of a failed consistency check now refuse to restart, making it more likely that the operator notices that there is a persistent issue in a timely manner. [#42149][#42149]
+- Client usernames can now be defined to start with a digit; in particular, all-digit usernames are now permitted. [#42989][#42989]
+- When the replicas within a range are corrupted, outliers are now terminated. Previously, the leaseholder replica would terminate, regardless of which replicas disagreed with each other. This is expected to curb the spreading of corrupted data better than the previous approach. [#42149][#42149]
```

Comment:
~Might be worth rephrasing to say what this is doing up front rather than at the end, e.g., "Curb the spread of corrupted data by terminating outliers when the replicas within a range are corrupted.  Previously, ... "